active/IndexPath/IndexPath.md

```diff
@@ -0,0 +1,96 @@
+# Background
+
+To easily track selection in nested collections, we need an object that we can use to uniquely identify an item in nested collections.
+This can be done with the IndexPath.
+
+# Description
+
+The `IndexPath` class provides indexing of nested collections, for example in a TreeView.
+
+# Examples
+
+In the tree below, using the following IndexPath 
+```c#
+
+var indexPath = IndexPath.CreateFromIndices(new List<int>{ 1, 0, 3 });
+
+```
+ would select the item "Kitchen cabinet style", since it is in the second group of first nesting level, first group in the second level of nesting and is the fourth element in the last nesting level.
+
+![Sample tree selection](./sample-tree-selection.png)
```

Comment:
Might just be on my end, but the image isn't rendering for me.

Reply:
I think the webview is not really aware of the relative path's here. Downloading them should work correctly.

Reply:
Weird, it's there, but it doesn't show up. I can vouch that the picture's accurate :)